### DISCUSSION TITLE
"Upload proof of your academic status" page stuck

### DISCUSSION THREAD

**Discussion by Chi-O** [community/community#67200] [GitHub Education]
### Hi everyone,

I am trying to renew my student benefits, however, I'm unable to get past the "Upload proof of your academic status" page. After uploading the photo of my proof of enrolment, I click "Process Application", the progress bar loads, and once it completes the page resets and I'm taken to the top of the page. 

I have tried this on Chrome and Edge on my laptop, as well as Safari on my iPad and iPhone but getting the same loop. 

Please let me know if you can assist with this.

Thank you!

**Comment by shreya-oeo** [comment]
Clear cache and try again.

**Comment by Chi-O** [comment]
I tried it with a different proof document and it works now.

**Reply by ManethNin** [reply]
I also got that issue. Thank you

**Comment by cgorlla** [comment]
Same as #67837
This seems to be an OCR issue where the text cannot be read, resizing the image to make the relevant text larger should work. Wasted an hour on this.

**Reply by MarcoMihalca** [reply]
I've tried, but mine still doesn't seem to work. I am stuck at Processing information... and I get error 413 in the console. I've tried multiple devices, times and different browsers as well :).

**Reply by dr-msr** [reply]
I got this error : 

Uncaught TypeError: data.responseJSON is undefined
Source map error: Error: request failed with status 404

**Reply by omeryagmurlu** [reply]
I get the same error.

**Reply by jeremiah-ang** [reply]
Hey. Got a similar issue where my photo of my staff card was not processed and was stuck at “processing application”. Uploaded my official emplacement letter instead and it was successfully processed. Likely because my staff Id didn't have a date on it whereas my official letter had.

**Reply by ndizihiwesimon** [reply]
On my end it loads and never leave the page, and i don't get any error, is it supposed to be redirected after submitting document(photo)? I am using Student ID

**Reply by MarcoMihalca** [reply]
Idk about that, but do make sure to actually TAKE the photo, not just upload it. That's what did it for me! :)

**Comment by dr-msr** [comment]
Hi, its been 6 month ago since I replied to this thread, sorry for the late but here's how it worked for me (if i remember correctly).

**Use your phone to do the verification, not desktop/PC. Actually snap the document using the camera from the web interface, not "select media/photo from gallery".**

Those who got it working using this tip, kindly response so that others can try this too, and maybe the dev would fix this.
